features/tracker_board_view/lib/src/composities/web_tracker_board_composite.dart

```diff
@@ -0,0 +1,133 @@
+import 'package:tracker_board_view/src/widgets/profile_menu/web_profile_menu.dart';
+import 'package:tracker_board_view/src/widgets/time_tracking_card/web_tracker_card.dart';
+
+class WebTimeTrackingBoardComposite extends StatelessWidget {
+  final TrackerBoardState state;
+  final User user;
+
+  final void Function() onLogout;
+
+  const WebTimeTrackingBoardComposite({
+    required this.state,
+    required this.user,
+    required this.onLogout,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final Map<String, PeriodActivities> mappedActivities = _groupByCategoryName(
+        PeriodActivities(
+            current: state.currentPeriodActivities,
+            previous: state.previousPeriodActivities));
+    final List<String> categoryNames = mappedActivities.keys.toList();
+    return Center(
+      child: SingleChildScrollView(
+        child: SingleChildScrollView(
+          scrollDirection: Axis.horizontal,
+          child: SizedBox(
+            height: 400,
+            child: Row(
+              mainAxisSize: MainAxisSize.min,
+              children: <Widget>[
+                Column(
+                  children: <Widget>[
+                    WebProfileMenu(
+                      avatar: const AssetImage(AppImages.jeremy),
+                      firstName: user.fullName,
+                      lastName: 'Popov',
+                      selectedPeriodTimeType: state.periodTimeType,
+                      onDailyPressed: () {
+                        BlocProvider.of<TrackerBoardBloc>(context,
+                                listen: false)
+                            .add(const PressDailyButton());
+                      },
+                      onMonthlyPressed: () {
+                        BlocProvider.of<TrackerBoardBloc>(context,
+                                listen: false)
+                            .add(const PressMonthlyButton());
+                      },
+                      onWeeklyPressed: () {
+                        BlocProvider.of<TrackerBoardBloc>(context,
+                                listen: false)
+                            .add(const PressWeeklyButton());
+                      },
+                    ),
+                    const SizedBox(height: AppDimensions.padding10),
+                    TextButton.icon(
+                      onPressed: onLogout,
+                      icon: const Icon(Icons.logout, color: Colors.white),
+                      label: Text(
+                        'general.logout'.tr(),
+                        style: AppTextStyle.rubicRegular20,
+                      ),
+                    )
+                  ],
+                ),
+                const SizedBox(width: AppDimensions.padding24),
+                Container(
+                  width: 700,
+                  height: 400,
+                  child: Wrap(
+                    direction: Axis.horizontal,
+                    children: <Widget>[
+                      ...categoryNames.map(
```

Comment:
Why do you need ... here, just make children: categoryNames.map().toList() smtn like that